composer.json

```diff
@@ -40,7 +40,7 @@
         "symfony/twig-bridge": "^2.3.5 || ^3.0",
         "symfony/validator": "^2.3 || ^3.0",
         "twig/extensions": "^1.0",
-        "twig/twig": "^1.26"
+        "twig/twig": "^1.28"
```

Comment:
Changing the version constraint of a dependency was never a BC break, @fabpot is right.

According to semver, changing dependency constraint is not a BC break, even for PHP constraint AFAIK.

After that, it depends of the choice of the maintainers.

For Sonata, ATM, we just want to lock constraints and change them on major only for PHP, Symfony (because of main dependency and LTS support), and when we want to change a constraint for a next major bump (e.g. Twig `^2.0` instead of `^1.28`).

So no, it's not a BC break for this (and our) case. :wink: 

Stop me if I'm wrong.